src/lib/webpack/run-webpack.js

```diff
@@ -38,7 +38,12 @@ const devBuild = async (env, onprogress) => {
 				let serverAddr = `${protocol}://${host}:${chalk.bold(port)}`;
 				let localIpAddr = `${protocol}://${ip.address()}:${chalk.bold(port)}`;
 
-				process.stdout.write(chalk.green('\nCompiled successfully!!\n\n'));
+				if (stats.hasErrors()) {
+					process.stdout.write(chalk.red('\Build failed!\n\n'));
+				} else {
+					process.stdout.write(chalk.green('\nCompiled successfully!\n\n'));
+				}
```

Comment:
It is now kinda different with @reznord's changes. Basically we would now show address only for successful builds and clear console after each build (IMHO that's much nicer).